packages/core/examples/popoverExample.tsx

```diff
@@ -232,18 +232,20 @@ export class PopoverExample extends BaseExample<IPopoverExampleState> {
                 <label className={Classes.LABEL}>
                     Enter some text
                     <input
-                        autoFocus={true}
+                        autoFocus={false}
                         className={Classes.INPUT}
                         type="text"
                     />
                 </label>
             </div>,
-            <Slider
-                min={0}
-                max={10}
-                onChange={this.handleSliderChange}
-                value={this.state.sliderValue}
-            />,
+            <div style={{ width: "300px" }}>
```

Comment:
yes i think that's the better solution. other inputs set their own widths so slider can follow suit. it'll make everyone's lives easier. there have been other issues reported with slider sizing (i think internally)